modules/administration/pages/tuning-changelogs.adoc

```diff
@@ -25,13 +26,13 @@ The task is run in the background by Taskomatic, so you can continue to use {pro
 You can delete cached data from the command line, using the SQL interface:
 
 ----
-spacewalk-sql -i
-SQL> delete from rhnPackageRepodata;
-SQL> insert into rhnRepoRegenQueue (id, CHANNEL_LABEL, REASON, FORCE)
-(select sequence_nextval('rhn_repo_regen_queue_id_seq'),
-        C.label,
-        'package summary modification',
-        'Y'
-   from rhnChannel C);
-SQL> \q
+# spacewalk-sql -i
+yourdatabase=# DELETE FROM rhnPackageRepodata;
+yourdatabase=# INSERT INTO rhnRepoRegenQueue (id, CHANNEL_LABEL, REASON, FORCE)
+               (SELECT sequence_nextval('rhn_repo_regen_queue_id_seq'),
+                       C.label,
+                       'package summary modification',
+                       'Y'
+                FROM rhnChannel C);
+yourdatabase=#  \q
 ----
```

Comment:
I guess that what happens here is in fact:
1. We remove the cached data
2. We add all packages to a queue so the cached data is regenerated.

Reply:
correct. I would also change 'package summary modification' into something more generic. AFAIK we copied this from a schema migration where the reason for re-generation was a change in the summary.

Reply:
References changed to "delete and regenerate", but I am not sure what you mean about:

> I would also change 'package summary modification' into something more generic.

(maybe because I am not so familiar with the DB schema)

Reply:
That string is just a text which should tell you why this job is there.
Not 100% sure if it is visible somewhere.

Reply:
Ah!

Reply:
Done.